sklearn/metrics/cluster/unsupervised.py

```diff
@@ -103,10 +124,39 @@ def silhouette_score(X, labels, metric='euclidean', sample_size=None,
             X, labels = X[indices].T[indices].T, labels[indices]
         else:
             X, labels = X[indices], labels[indices]
-    return np.mean(silhouette_samples(X, labels, metric=metric, **kwds))
-
-
-def silhouette_samples(X, labels, metric='euclidean', **kwds):
+    return np.mean(silhouette_samples(X, labels, metric=metric,
+                                      block_size=block_size, **kwds))
+
+
+def _process_block(X, labels, start, block_n_rows, block_range, add_at,
+                   label_freqs, metric, kwds):
```

Comment:
Just a niptick : _process_block -> _silhouette_process_block.
I know it's a private function but maybe it will be easier for comprehension if you put a little docstring of the arguments ???